pages/common/kind.md

```diff
@@ -0,0 +1,21 @@
+# kind
+
+> Kind is a tool for running local Kubernetes clusters using Docker container "nodes".
+> Kind was primarily designed for testing Kubernetes itself, but may be used for local development or CI.
+> More information: <https://github.com/kubernetes-sigs/kind>.
+
+- Create a local Kubernetes cluster:
+
+`kind create cluster --name {{cluster-name}}`
+
+- Delete one or more clusters:
+
+`kind delete clusters {{cluster-name}}`
+
+- Get one of [clusters, nodes, kubeconfig]:
+
+`kind get clusters`
```

Comment:
How does one determine which one to fetch? If it's by the keyword there, what about this?
```suggestion
- Get details about clusters, nodes, or the kubeconfig:

`kind get {{clusters|nodes|kubeconfig}}`
```